locale/lang-tools/add_string.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import argparse
+import toml
+from glob import glob
+from merge import unmerge
+
+def copy_template_internal(res, template):
+    for key in template:
+        if key not in res and key != 'ALERTS':
```

Comment:
do we actually need the 'ALERTS' key? I don't understand why we need a "double" key for alerts

Reply:
right then this can be changed in a future PR